### PR TITLE
Fix values shown on charts when the collector is running in entry mode.

### LINKED ISSUE #159: Fix bytes charts in entry mode
When we are using entry mode, the charts that shows bytes written and read are not showing nothing.

### LINKED ISSUE #161: Fix `apps.task_close`
We are removing on kernel-ring the information on hash table, but this information is necessary on user-ring, so we need to remove there and not here.

### LINKED ISSUE #162: Fix thread_create
The threads counts for PIDs are happen only on `return` mode, it is necessary to move part of the code.

### PR DESCRIPTION
**Pull request by thiagoftsm**
Fixes #159
Fixes #161 
Fixes #162 

Change value read to create charts that gives bytes information.
Bring a new variable and also comments the code that is creating problems.

### CODE REVIEW

File: kernel/netdata_ebpf.h [hunk 1]
```diff
@@ -46,6 +46,8 @@ struct netdata_pid_stat_t {
     __u32 fork_err;
     __u32 clone_err;
     __u32 close_err;
+
+    __u8 removeme;
```
Comment (underhood) [inline kernel/netdata_ebpf.h:50]: ?

**Comment by prologic** [comment]
@thiagoftsm 

> Fix charts

Can you elaborate a bit? This makes for a hard to understand commit message :) (_unless they were just completely broken!_)

**Comment by thiagoftsm** [comment]
> @thiagoftsm
> 
> > Fix charts
> 
> Can you elaborate a bit? This makes for a hard to understand commit message :) (_unless they were just completely broken!_)

Sorry I was tired yesterday.

File: kernel/netdata_ebpf.h (hunk 1 above)
Comment (thiagoftsm) [reply kernel/netdata_ebpf.h:50]: When we remove in `kernel-ring` the values from hash-table using `bpf_map_delete_elem`, we are not able to read the data on `user-ring`, I created this variable  that acts a flag, it is set on `kernel-ring` to be removed on `user-ring`.

**Pull request merged**